extensions/proposals/OES_EGL_image_external/extension.xml

```diff
@@ -44,6 +45,13 @@
 
   <idl xml:space="preserve">
 [NoInterfaceObject]
+interface WebGLVideoFrameInfo {
+  readonly attribute double currentTime;
+  readonly attribute unsigned long textureWidth;
+  readonly attribute unsigned long textureHeight;
+};
```

Comment:
@RafaelCintron Thanks for the review. It is added to notify application what currently WebGL is rendering of which frame of Video. Application can detect the delaying time and adjust audio synchronization etc.